src/components/z-menu/index.tsx

```diff
@@ -1,4 +1,5 @@
 import {Component, Element, Event, EventEmitter, Host, Listen, Prop, State, Watch, h} from "@stencil/core";
+import {DividerOrientation, KeyboardCode} from "../../beans";
 
```

Comment:
non c'entra con questa riga ma, già che la facciamo breaking, mi è venuto in mente che sarebbe molto comodo avere la prop `floating` di default a `true`: ancora non ho visto nessun prodotto che la usa a `false` spingendo giù la pagina quando viene aperto il menu 😂